data/test/scenarios/events-test_actions.cfg

```diff
@@ -0,0 +1,89 @@
+
+# This is kinda a duplicate of a basic unit test system sanity test, but... it seems to fit here as well...
+{GENERIC_UNIT_TEST "event_test_action_wml" (
+    [event]
+        name=start
+        {SUCCEED}
+    [/event]
+)}
+
+# Test that dynamically adding an event from Lua works
+# There's two different ways of adding events, so test both
```

Comment:
I acknowledge these were written before @Pentarctagon's new style for test comments, but please update the test files to the  `API(s) being tested:` style.

A `# wmllint: no translatables` would be good too.